### PR TITLE
update/add SwiftNIO family projects + Swift Crypto

### PR DESCRIPTION
**Pull request by weissi**
Just updating/adding some SwiftNIO family projects & Swift Crypto
.

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

**Review by shahmishal — APPROVED** [review]
Thanks!

**Review by clackary — APPROVED** [review]
Thanks @weissi!

**Comment by weissi** [comment]
@shahmishal sorry to request re-review, I added another version & package.

**Comment by weissi** [comment]
@swift-ci test

**Review by clackary — APPROVED** [review]

**Review by tomerd — APPROVED** [review]

File: projects.json [hunk 1]
```diff
@@ -2878,7 +2878,7 @@
     "repository": "Git",
     "url": "https://github.com/apple/swift-nio",
     "path": "swift-nio",
-    "branch": "master",
+    "branch": "main",
```
Comment (tomerd) [inline projects.json:2881]: @shahmishal how does the interaction between the branch and the commit hashes work? should this be based on a tag?

Comment (shahmishal) [reply projects.json:2881]: All of the CI infrastructure use hash, not the branch info. We added this option to be able to know development branch for the project, normally it's main/master but sometime people use different branch name.

**Review by ktoso — APPROVED** [review]

**Pull request merged**